java/src/jni/h5aImp.c

```diff
@@ -1123,16 +1121,18 @@ Java_hdf_hdf5lib_H5_H5AreadVL(JNIEnv *env, jclass clss, jlong attr_id, jlong mem
             H5_LIBRARY_ERROR(ENVONLY);
 
         /* Cache class types */
-        jclass cBool   = ENVPTR->FindClass(ENVONLY, "java/lang/Boolean");
+        /* jclass cBool   = ENVPTR->FindClass(ENVONLY, "java/lang/Boolean"); */
```

Comment:
Why are these commented out?

Reply:
They aren't currently used because the boolean type handling code is commented out. I think I asked @byrnHDF about this, but I can't remember how the boolean stuff is handled.